src/learn/dotnet-two/index.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Dependency injections and interfaces in Dotnet
+description: Learn how to use dependency injections and interfaces in Dotnet
+tags:
+  - workshop
+keywords:
+  - DOTNET
+  - ASP.NET 
+  - CSharp
+challenge: https://github.com/foundersandcoders/DOTNET-Workshop-Two/blob/main/README.md
+---
+
+
+
+## Setting up .Net
```

Comment:
I reckon maybe search the whole repo for dotnet, .net (case insensitive) and change them all to .NET. 

maybe not every single one because you might be referring to CLI package but probably most of them.